xlators/features/quota/src/quota.c

```diff
@@ -1048,10 +1048,7 @@ quota_check_limit_continuation(struct list_head *parents, inode_t *inode,
         goto out;
     }
 
-    list_for_each_entry(entry, parents, next)
-    {
-        parent_count++;
-    }
+    list_for_each_entry(entry, parents, next) { parent_count++; }
```

Comment:
Any reason for changing this ?

Reply:
> Any reason for changing this ?

Oh, I see it's because of the clang-format recommendation...

We never put the body of a loop in the same line. Unfortunately clang-format changes how it does format the code when the version is changed. Recently it's recommending many unwanted changes. Probably we'll need to revisit its configuration.

@zellong if possible, could you remove this change and just submit the required modification (your original patch) ?